packages/devtools/web/styles_dark.css

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2019 The Chromium Authors. All rights reserved. */
+/* Use of this source code is governed by a BSD-style license that can be */
+/* found in the LICENSE file. */
+
+/* If you add a CSS variable here, be sure to also add it to styles.css */
+:root {
+    --border-color-dark: #484848;
+    --border-color: #484848;
+    --counter-color: #999;
+    --dark-shadow-color: #999;
+    --default-background-rgb: 30, 30, 30;
+    --default-background: #1e1e1e;
```

Comment:
Would something like this work to avoid duplicated colours for those we have both RGB/Hex?

```css
    --default-background-rgb: 30, 30, 30;
    --default-background: rgba(var(default-background-rgb), 1);
```

Reply:
good idea. done.